gtsfm/bundle/bundle_adjustment.py

```diff
@@ -136,6 +145,37 @@ def create_computation_graph(self, sfm_data_graph: Delayed) -> Delayed:
         return dask.delayed(self.run)(sfm_data_graph)
 
 
+def aggregate_ba_stats(ba_data: GtsfmData) -> Dict[str,Any]:
```

Comment:
Maybe change Any to Dict[str, float]?

Reply:
technically it is `Dict[str,Union[float,Dict[str,float]]]` but i thought that was a bit messy. the output is nested, so technically it is not `Dict[str, float]`

Reply:
`Any` works for me